lib/protocols.js

```diff
@@ -1039,6 +1054,24 @@
     }
   }());
 
+  var ProtobufRoot = (function() {
+    var protobufRoots = [];
```

Comment:
we're offering the possibility to register multiple roots, but this isn't a requirement. Does this add any advantage? How about having just one to speed up decoding and simplify code?

Reply:
Yeah, Its not a requirement. I will remove this.